packages/babel-plugin-minify-constant-folding/src/replacements.js

```diff
@@ -51,7 +58,7 @@ module.exports = ({ types: t }) => {
               return evaled.value;
             })
             .join(sep.value);
-          return bad ? undefined : t.stringLiteral(str);
+          return bad ? void 0 : t.stringLiteral(str);
```

Comment:
Why does this use `void 0` instead of `undefined`?

Reply:
we use `void 0` everywhere in the project. Just sticking to that.